src/view.rs

```diff
@@ -717,3 +716,43 @@ pub fn view_grid_grammar(m: &Model, coord: &Coordinate, sub_coords: Vec<Coordina
         </div>
     }
 }
+
+pub fn view_context_menu(m: &Model) -> Html {
```

Comment:
perhaps a better name instead of using for readability.

Reply:
better name such as...? i think this one is pretty explanatory.